api/modules/crudRefactorClean.js

```diff
@@ -0,0 +1,316 @@
+        )
+    }
+    const createTopics = (params) => {
+        const {
+            title,
+            order,
+            rich_text_content,
+            created_on,
+            course_id,
+        } = params.values;
+
+        Topic.create(
+            {
+                title,
+                order,
+                rich_text_content,
+                created_on,
+                course_id,
+            }
+        )
+        .then(topics => {
+            console.log('Id of new topic:', topics.id);
+        });
+
+    } 
+    const deleteTopics = (params) => {
+        const { topicId } = params 
+        return (
+            Topic.destroy({
+                where: {
+                    id: topicId
+                }
+            })
+            .then(topics => {
+                console.log('Course deleted correctly');
+                return topics
+            })
+        )
+    }
+    const fetchTopicsByLanguage = (params) => {
+        const { languageId } = params 
+        return (
+            Topic.findAll({
+                where: {
+                    language_id: languageId
+                }
+            })
+            .then(topics => {
+                console.log('Topics:', JSON.stringify(courses, null, 4));
+                return topics
+            })
+        )
+    }
+    //USERS
+    const fetchUsers = (params) => {
+        return (
+            User.findAll({
+            })
+            .then(users => {
+                console.log('All Topics:', JSON.stringify(users, null, 4));
+                return users
+            })
+        )
+    }
+    const createUsers = (params) => {
+        const {
+            email,
+            hashed_password,
+            date_registered,
+            google_id,
+        } = params.values;
+
+        User.create(
+            {
+                email,
+                hashed_password,
+                date_registered,
+                google_id,
+            }
+        )
+        .then(users => {
+            console.log('Id of new user:', users.id);
+        });
+
+    } 
+    const deleteUsers = (params) => {
+        const { userId } = params 
+        return (
+            User.destroy({
+                where: {
+                    id: userId
+                }
+            })
+            .then(users => {
+                console.log('Course deleted correctly');
+                return users
+            })
+        )
+    }
+    //FEEDBACK
+    const fetchFeedback = (params) => {
+        return (
+            Feedback.findAll({
+            })
+            .then(feedbacks => {
+                console.log('All feedbacks:', JSON.stringify(feedbacks, null, 4));
+                return feedbacks
+            })
+        )
+    }
+    const createFeedback = (params) => {
+        const {
+            topic_id,
+            user_id,
+            text,
+            date_created,
+            adapter_direction
+        } = params.values;
+
+        Feedback.create(
```

Comment:
wrap in a return statement (this will be important when trying to call this function from another module)